spec/models/miq_expression_spec.rb

```diff
@@ -1240,76 +1197,167 @@
         expect(MiqExpression.normalize_date_time("This Quarter", "UTC").utc.to_s).to eq("2011-01-01 00:00:00 UTC")
         expect(MiqExpression.normalize_date_time("This Quarter", "UTC", "end").utc.to_s).to eq("2011-03-31 23:59:59 UTC")
       end
+    end
+  end
+
+  describe ".to_human" do
+    context "generates string representation of expression atoms that do/don't have alias keys" do
```

Comment:
This context looks like an expectation, and the expectations below it look like contexts. Do you think this could be rearranged a little to provide better documentation (try running with the `--format doc` flag to see what this outputs). Something like:

```
MiqExpression#to_human
  generates a human readable string for a field type expression
  generates a human readable string for a field type expression with an alias
# etc...
  when given values with relative dates/times
    generates a human readable string with # etc, etc..
```